providers/AVISO.json

```diff
@@ -123,6 +123,83 @@
         }
     },
     "elevation": {
+        "FES2012": {
+            "format": "FES-netcdf",
+            "constituents": [
+                "t2",
+                "msf",
+                "mtm",
+                "l2",
+                "mm",  
+                "q1",
+                "2n2",
+                "n4",
+                "o1",
+                "m3",
+                "n2",
+                "k1",
+                "s1",
+                "ms4",
+                "r2",
+                "mu2",
+                "eps2",
+                "s4",
+                "ssa",
+                "l2",
+                "mn4",
+                "m2",
+                "j1",
+                "m6",                
+                "mf",
+                "m4",
+                "p1",  
+                "k2",
+                "mks2",
+                "s2",
+                "m8",   
+                "nu2"
+            ],
+            "model_file": [
```

Comment:
The one file I couldn't find a match for was `Z0_FES2012_SLEV.nc` - have left it out here for now